meson.build

```diff
@@ -43,6 +50,19 @@ if cc.get_define('__ANDROID__') != ''
              'src/unix/pthread-fixes.c' ]
 endif
 
+# OSX systems
+if cc.get_define('__APPLE__') != ''
+    uvsrc += [
+    	'src/unix/bsd-ifaddrs.c',
+	'src/unix/darwin.c',
+	'src/unix/darwin-proctitle.c',
```

Comment:
This looks like a mix of tabs and spaces, please `:retab`.

Reply:
Should be tab-free now.